src/components/basics/FilterMenu.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react';
+import {
+  Button,
+  Checkbox,
+  Radio,
+  Icon,
+  Link,
+  ListItem,
+  styles,
+  TooltipLinkList,
+  WithTooltip,
+} from '@storybook/design-system';
+import { styled } from '@storybook/theming';
+
+import { ToggleButton } from './ToggleButton';
+
+export type Items = Array<Omit<React.ComponentProps<typeof ListItem>, 'LinkWrapper'>>;
+export type Value = Array<string>;
+
+export type FilterMenuProps = Omit<
+  React.ComponentProps<typeof WithTooltip>,
+  'children' | 'tooltip' | 'trigger'
+> & {
+  disabled?: boolean;
+  includeClearButton?: boolean;
+  items: Items;
+  label: string;
+  multiple?: boolean;
+  onChange: (value: Value) => void;
+  TriggerButton?: React.ComponentType<React.ComponentProps<typeof Button> & { active: boolean }>;
+  useActiveStyles: boolean;
+  value: Value;
+};
+
+const ClearButton = styled((props) => <Link isButton {...props} />)`
+  border-top: 1px solid ${styles.color.border};
+  display: block;
+  font-weight: ${styles.typography.weight.bold};
+  line-height: 18px;
+  padding: 7px 15px;
+  text-align: center;
+  width: 100%;
+  &:hover {
+    transform: none;
+  }
+`;
+
+const Trigger = styled(({ active, Component, children, ...props }) => (
+  <Component active={active} ButtonWrapper="span" {...props}>
+    {children}
+    <Icon icon="arrowdown" />
+  </Component>
+))`
+  & svg {
+    margin-left: 6px;
+    margin-right: 0;
+  }
+`;
+
+export const FilterMenu: React.VFC<FilterMenuProps> = ({
```

Comment:
Is this the same as https://github.com/storybookjs/component-catalog/blob/main/components/FilterMenu.tsx from the showcase? Does it make sense to move it into components-marketing?

Reply:
It's almost the exact same! I could certainly merge the two into FilterMenu